packages/frontend/src/ee/features/customRoles/utils/roleValidation.ts

```diff
@@ -0,0 +1,34 @@
+import { isSystemRole } from '@lightdash/common';
+
+/**
+ * Validates a role name
+ * @param value - The role name to validate
+ * @returns Error message if invalid, null if valid
+ */
+export const validateRoleName = (value: string): string | null => {
```

Comment:
don't we have the same validation in the backend ? perhaps we should migrate this to common, and reuse